src/gt4py/backend/pyext_builder.py

```diff
@@ -133,8 +133,11 @@ def build_gtcpu_ext(
     clean=False,
     debug_mode=False,
     add_profile_info=False,
+    extra_include_dirs: list = None,
 ):
     include_dirs = [gt_config.build_settings["boost_include_path"]]
+    if extra_include_dirs:
+        include_dirs.extend(extra_include_dirs)
```

Comment:
We should have this for `build_gtcuda_ext` as well.

Reply:
Done